nomos-da/network/subnetworks-assignations/src/versions/v1.rs

```diff
@@ -40,7 +42,7 @@ impl FillFromNodeList {
 }
 
 impl MembershipHandler for FillFromNodeList {
-    type NetworkId = u16;
+    type NetworkId = u32;
```

Comment:
I thought we said `u16` maybe we need to change the SubnetworkId (can be done later on).
There is some job to stabilize what types do we use. I see you have already pierce through the `BlobId` nightmare we had :wink:

Reply:
So we have `ColumnIdx` which goes into the blob and is used by the storage. Then we have `SubnetworkId`, which is used by the protobufs. We can either have one type or two. Atm SubnetworkId is u32 because that maps conveniently into the protobuf type (where it will be uaed the most). Either way we'll need to make a conversion somewhere.